DataFormats/Detectors/MUON/MID/include/DataFormatsMID/Track.h

```diff
@@ -143,19 +143,28 @@ class Track
   /// Gets the word allowing to compute the chamber efficiency
   uint32_t getEfficiencyWord() const { return mEfficiencyWord; }
 
-  /// Sets the fired local board for efficiency calculation
-  /// \param locId local board ID in the range 1-234
-  void setFiredLocalBoard(int locId) { setEfficiencyWord(8, 0xFF, locId); }
+  /// Sets the fired FEE ID
+  void setFiredFEEId(int uniqueFeeId) { setEfficiencyWord(8, 0x7FFF, uniqueFeeId); }
+
+  /// Gets the fired FEE ID
+  int getFiredFEEId() const { return (mEfficiencyWord >> 8) & 0x7FFF; }
 
   /// Gets the fired local board for efficiency calculation
+  /// FIXME: Obsolete: remove after changing QC
```

Comment:
Note that instead of a FIXME (that might be forgotten...) you might prefer to use https://en.cppreference.com/w/cpp/language/attributes/deprecated that will result in actual warnings (that might be ignored, I get it :wink: )

Reply:
I did not know about this attribute. I'll give it a try